src/Exporter.php

```diff
@@ -57,7 +57,7 @@ public function getResponse(string $format, string $filename, SourceIteratorInte
         }
         $writer = $this->writers[$format];
 
-        $callback = function () use ($source, $writer) {
+        $callback = function () use ($source, $writer): void {
```

Comment:
This is needed? oO

Reply:
If you mean "this works?", yes it works :) (why not?). If you mean "Is this necessary?" none of this PR is, it just makes things safer.

Reply:
I can understand this stuff for the api methods (protected and public things). But for internal functions? I don't think that this is useful at all.

Reply:
It doesn't hurt though, and I use php-cs-fixer for this, and I'd hate to make an exception for this kind of stuff each time I run it. Also I'd argue it's good to have type hinting for internal functions: it helps your IDE understanding what happens, and if you do wrong things it crashes early.